liquibase-core/src/main/java/liquibase/command/core/InternalDropAllCommandStep.java

```diff
@@ -100,6 +100,7 @@ public void run(CommandResultsBuilder resultsBuilder) throws Exception {
                     commandScope.getArgumentValue(DATABASE_ARG).dropDatabaseObjects(schema);
                 }
             } catch (LiquibaseException liquibaseException) {
+                log.severe("Error occurred during dropAll: " + liquibaseException.getMessage());
```

Comment:
Good idea. Can you also include the raw exception in the log message so we get a stack trace in the logs?

```suggestion
                log.severe("Error occurred during dropAll: " + liquibaseException.getMessage(), liquibaseException);
```

Reply:
Yes, that makes sense.